### PR TITLE
updating travis schema

### PR DESCRIPTION
**Pull request by jGaboardi**
Addressing #15

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,24 +1,19 @@
...
   - PYSAL_PYPI=false MPLBACKEND='pdf'
 
 matrix:
+  fast_finish: true
   allow_failures: #allow travis tests to fail if using the github version of libpysal
-      - python: 3.6
-        env: PYSAL_PYPI=false MPLBACKEND='pdf'
-      - python: 3.7
-        env: PYSAL_PYPI=false MPLBACKEND='pdf'
+      - name: "python-3.6"
```
Comment (jGaboardi) [inline .travis.yml]: **Note:**
 Should each version of Python in the matrix have a `PYSAL_PYPI=true` and a `PYSAL_PYPI=false`?

Comment (jGaboardi) [reply .travis.yml]: **Note:**
How to accommodate `allow_failures` and `include` to perform all combinations in testing matrix?

**Comment by jGaboardi** [comment]
This PR *may* be good to go now for the adoption of [the new Travis CI testing procedure](https://github.com/pysal/spopt/issues/15). The method for specifying the language and version seems a bit alien to me and also raises questions about how to implement this process with the need for `allow_failures` as will be [the case with `spaghetti`](https://github.com/pysal/spaghetti/blob/a177de609070b5e3f655660895b8ed03e5a23fd4/.travis.yml#L19) when I start with pysal/spaghetti#299. However, all this is a bit conjecture at the moment because [we aren't actually running any tests yet...](https://travis-ci.com/pysal/spopt/jobs/227973005#L792)

**Comment by knaaptime** [comment]
superseded by #20

**Comment by sjsrey** [comment]
We moved to pytest so closing ths.

**Pull request closed without merging**